app/config/security.yml

```diff
@@ -9,6 +9,22 @@ security:
             entity: { class: AppBundle:User, property: username }
 
     firewalls:
+        user_area:
+            pattern: ^/profil
```

Comment:
Nerozumiem prečo, ale nefunguje mi to ako by malo. Ak sa aj prihlásim, tak ma po vstupe na /profil presmeruje na login page.